src/pyflex/window_selector.py

```diff
@@ -406,6 +433,11 @@ def determine_signal_and_noise_indices(self):
         if self.config.signal_start_index is None:
             self.config.signal_start_index = self.config.noise_end_index
 
+        logger.info("Noise start index and noise end index: %d, %d"
+                    %(self.config.noise_start_index, self.config.noise_end_index))
+        logger.info("Signal start index and noise end index: %d, %d"
+                    %(self.config.signal_start_index, self.config.signal_end_index))
+
```

Comment:
True. I forgot about that. It might be nicer for users to read something in seconds, e.g.:

```
Noise from 0-113 seconds, signal from 113-1051 seconds.
```